### ISSUE TITLE
Info component Toolbar bug

### ISSUE THREAD

**Issue by sheepyhollow** [zentrumnawi/solid-frontend#1568]
In der Info-component gibt es einen scroll-bug:

![image](https://user-images.githubusercontent.com/13869236/174457309-1eae929f-cf40-4e51-ae0f-9a81631ef196.png)

Ich dachte, dass wir das schon lange gefixt haben, aber vielleicht ist es beim Einbau der Settings passiert.

**Comment by sheepyhollow** [comment]
@lthuong: Ich würde gerne #1563 abschließen - dieses Issue kann auch später in einen anderen PR.

**Comment by sheepyhollow** [comment]
Ich habe noch etwas in [info.component.css](https://github.com/zentrumnawi/solid-frontend/blob/dev/libs/solid/skeleton/src/lib/components/info/info.component.scss) herumprobiert. Ich habe es fast hinbekommen, aber mit `position: fixed` verschwindet der mat-table-header-paginator für kleine Bildschirme-

```
.container {
  height: 100%;
  overflow: hidden;

  mat-tab-group {
    ::ng-deep {
      .mat-tab-header {
        box-shadow: 0px 4px 2px -2px rgb(0 0 0 / 20%);
        height: 56px;
        border-bottom: 0;
        background-color: white;
        margin-left: 60px;
        position: fixed;
        z-index: 999;
      }
      .mat-tab-label {
        height: 56px;
      }
      .mat-tab-body-wrapper {
        position: relative;
        margin-top: 40px;
      }
    }
  }
}
.info-container {
  overflow-y: auto;
  height: 100%;
  z-index: -999;

  .button-back-container {
    background-color: white;
    box-shadow: 0px 4px 2px -2px rgb(0 0 0 / 20%);
    height: 56px;
    width: 64px;
    position: absolute;
    z-index: 99;

    .button-back {
      position: absolute;
      left: 16px;
      top: 8px;
    }
  }
}
```

**Comment by sheepyhollow** [comment]
Es ist kein Problem mit `position: fixed`, sondern mit `width`.

**Comment by sheepyhollow** [comment]
Gefixt mit `width: calc(100% - 56px)` in [6d42766](https://github.com/zentrumnawi/solid-frontend/pull/1563/commits/6d42766f46f439a85cbd22b3074e6785761f9f5b)

Ich verstehe zwar nicht, wieso 56px, denn die Breite des paginators ist 32px - aber so weit ich das sehen kann, passt es jetzt bei allen Bildschirmgrößen.

Der `z-index` ist auch noch wichtig damit die Toolbar oben liegt.

**Comment by lthuong** [comment]
Also eigentlich muss es 60px sein weil ich sehe, dass `margin-left: 60px` ist. Aber wenn es 60px ist, gibt es dann eine kleine Lücke am Ende des Headers, wahrscheinlich weil die `padding-right` von des paginators 4px. Dann ist 56px der richtige Wert

**Comment by sheepyhollow** [comment]
Ah! Makes total sense 😄

**Comment by lthuong** [comment]
D.h dass wir diese Issue auch abschließen können oder ?

**Comment by sheepyhollow** [comment]
Yes! Ah, ich dachte, ich hatte das automatisch per Commit message gefixt.
